docs/SUMMARY.md

```diff
@@ -40,6 +40,7 @@
 * [The `earthly` command](earthly-command/earthly-command.md)
 * [Configuration reference](earthly-config/earthly-config.md)
 * [CI integration](./ci-integration.md)
+* [Experimental features](experimental-features/features.md)
```

Comment:
Wondering if this should go under the Earthfile section rather?

Reply:
I was torn about that too, as some of the features aren't exactly related to Earthfiles, but I'm fine moving them there if we want to limit the number of top-level side links on the summary.

Reply:
I just moved it.

Reply:
Forgot to update SUMMARY too?

Reply:
good catch :)